server.js

```diff
@@ -1,5 +1,5 @@
 const express = require("express");
-const { Users } = require("./models");
+const { Users, Comments } = require("./models");
```

Comment:
Doesn't 'Comments' need to be it's on declared variable? 

ex: const { Comments } = require("./models");

Lemme know what you think.